challenges/objects-arrays.js

```diff
@@ -58,12 +81,18 @@ The resulting contact information should have a space between the first name and
 Name email@example.com
 
 Log the result of your new array. */
-const contactInfo = [];
+const contactInfo = graduates.map((item => {
+  return `${item.first_name} ${item.email}`;
+}));
 console.log(contactInfo);
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
-const uni = [];
+const uni = graduates.filter(item => {
+  if(item.university.includes("Uni")){
+    return item.university;
+  }
+})
```

Comment:
.filter() returns an element based on a true/false statement so there's no need for the if statement: 
```
const uni = graduates.filter(item => {
    return item.university.includes("Uni");
});
```
You also could have used the universities variable you declared earlier